zdm/iteration.py

```diff
@@ -313,7 +319,7 @@ def update_grid(grid,pset,survey):
         grid.rates=grid.pdv*grid.sfr_smear #does pdv mult only, 'by hand'
     
 
-def calc_likelihoods_1D(grid,survey,pset,doplot=False,norm=True,psnr=False,Pn=True,dolist=0):
+def calc_likelihoods_1D(grid,survey,lC,doplot=False,norm=True,psnr=False,Pn=True,dolist=0):
```

Comment:
right, because the grid knows its own pset now. Understood. Might see if there is some way to separate out the lC normalisation step in future, but for now we'll leave this floating here. Might not want to ither. probably not worth the mental effort right now.